### PR TITLE
Removed 5 from HTML

### PR DESCRIPTION
**Pull request by estelle**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ browser-compat: api.Gamepad
 ---
 {{DefaultAPISidebar("Gamepad API")}}
 
-HTML5 introduced many of the necessary components for rich, interactive game development. Technologies like `<canvas>`, WebGL, `<audio>`, and `<video>`, along with JavaScript implementations, have matured to the point where they can now support many tasks previously requiring native code. The Gamepad API is a way for developers and designers to access and use gamepads and other game controllers.
+HTML provies the necessary components for rich, interactive game development. Technologies like `<canvas>`, WebGL, `<audio>`, and `<video>`, along with JavaScript implementations support tasks that provide similar, if not the same, features as native code. The Gamepad API is a way for developers and designers to access and use gamepads and other game controllers.
```
Comment (teoli2003) [inline files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md:15]:
```suggestion
HTML provides the necessary components for rich, interactive game development. Technologies like `<canvas>`, WebGL, `<audio>`, and `<video>`, along with JavaScript implementations, support tasks that provide similar, if not the same, features as native code. The Gamepad API allows developers and designers to access and use gamepads and other game controllers.
```

File: files/en-us/web/api/html_drag_and_drop_api/file_drag_and_drop/index.md [hunk 2]
```diff
@@ -95,4 +95,4 @@ function dragOverHandler(ev) {
 
 - [HTML Drag and Drop API](/en-US/docs/Web/API/HTML_Drag_and_Drop_API)
 - [Drag Operations](/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Drag_operations)
-- [HTML5 Living Standard: Drag and Drop](https://html.spec.whatwg.org/multipage/interaction.html#dnd)
+- [ HTML Living Standard: Drag and Drop](https://html.spec.whatwg.org/multipage/interaction.html#dnd)
```
Comment (teoli2003) [inline files/en-us/web/api/html_drag_and_drop_api/file_drag_and_drop/index.md:98]:
```suggestion
- [HTML Living Standard: Drag and Drop](https://html.spec.whatwg.org/multipage/interaction.html#dnd)
```

File: files/en-us/web/api/html_drag_and_drop_api/index.md [hunk 3]
```diff
@@ -251,5 +251,5 @@ As can be seen in the [DataTransferItem interface's Browser Compatibility table]
 - [Drag Operations](/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Drag_operations)
 - [Dragging and Dropping Multiple Items](/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Multiple_items)
 - [Recommended Drag Types](/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Recommended_drag_types)
-- [HTML5 Living Standard: Drag and Drop](https://html.spec.whatwg.org/multipage/interaction.html#dnd)
+- [ HTML Living Standard: Drag and Drop](https://html.spec.whatwg.org/multipage/interaction.html#dnd)
```
Comment (teoli2003) [inline files/en-us/web/api/html_drag_and_drop_api/index.md:254]:
```suggestion
- [HTML Living Standard: Drag and Drop](https://html.spec.whatwg.org/multipage/interaction.html#dnd)
```

File: files/en-us/web/api/htmlanchorelement/index.md [hunk 4]
```diff
@@ -90,7 +90,7 @@ _Inherits methods from its parent, {{domxref("HTMLElement")}}._
 - {{domxref("HTMLAnchorElement.toString()")}}
   - : Returns a string containing the whole URL. It is a synonym for {{domxref("HTMLAnchorElement.href")}}, though it can't be used to modify the value.
 
-The `blur()` and `focus()` methods are inherited from {{domxref("HTMLElement")}} from HTML5 on, but were defined on `HTMLAnchorElement` in DOM Level 2 HTML and earlier specifications.
+The `blur()` and `focus()` methods are inherited from {{domxref("HTMLElement")}}. They were originally defined on`HTMLAnchorElement` in DOM Level 2 HTML and earlier specifications.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlanchorelement/index.md:93]: I think we can get rid of this historical information…
```suggestion
The `blur()` and `focus()` methods are inherited from {{domxref("HTMLElement")}}.
```

File: files/en-us/web/api/htmlelement/input_event/index.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ The **`input`** event fires when the `value` of an {{HTMLElement("input")}}, {{H
...
 
-For `<input>` elements with `type=checkbox` or `type=radio`, the `input` event should fire whenever a user toggles the control, per [the HTML5 specification](https://html.spec.whatwg.org/multipage/input.html#the-input-element:event-input-2). However, historically this has not always been the case. Check compatibility, or use the {{domxref("HTMLElement/change_event", "change")}} event instead for elements of these types.
+For `<input>` elements with `type=checkbox` or `type=radio`, the `input` event should fire whenever a user toggles the control, per [the HTML Living Standard specification](https://html.spec.whatwg.org/multipage/input.html#the-input-element:event-input-2). However, historically this has not always been the case. Check compatibility, or use the {{domxref("HTMLElement/change_event", "change")}} event instead for elements of these types.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlelement/input_event/index.md:26]:
```suggestion
For `<input>` elements with `type=checkbox` or `type=radio`, the `input` event should fire whenever a user toggles the control, per the [HTML Living Standard specification](https://html.spec.whatwg.org/multipage/input.html#the-input-element:event-input-2). However, historically this has not always been the case. Check compatibility, or use the {{domxref("HTMLElement/change_event", "change")}} event instead for elements of these types.
```

File: files/en-us/web/api/htmltextareaelement/index.md [hunk 6]
```diff
@@ -253,7 +253,7 @@ The **`HTMLTextAreaElement`** interface provides special properties and methods
   </tbody>
 </table>
 
-The two properties `tabIndex` and `accessKey` are inherited from {{domxref("HTMLElement")}} from HTML5 on, but were defined on `HTMLTextAreaElement` in DOM Level 2 HTML and earlier specifications.
+The two properties `tabIndex` and `accessKey` are inherited from {{domxref("HTMLElement")}}. They were originally defined on `HTMLTextAreaElement` in DOM Level 2 HTML and earlier specifications.
```
Comment (teoli2003) [inline files/en-us/web/api/htmltextareaelement/index.md:256]:
```suggestion
The two properties `tabIndex` and `accessKey` are inherited from {{domxref("HTMLElement")}}.
```

File: files/en-us/web/api/htmltextareaelement/index.md [hunk 7]
```diff
@@ -323,7 +323,7 @@ The two properties `tabIndex` and `accessKey` are inherited from {{domxref("HTML
   </tbody>
 </table>
 
-The two methods `blur()` and `focus()` are inherited from {{domxref("HTMLElement")}} from HTML5 on, but were defined on `HTMLTextAreaElement` in DOM Level 2 HTML and earlier specifications.
+The two methods `blur()` and `focus()` are inherited from {{domxref("HTMLElement")}}. They were originally defined on `HTMLTextAreaElement` in DOM Level 2 HTML and earlier specifications.
```
Comment (teoli2003) [inline files/en-us/web/api/htmltextareaelement/index.md:326]:
```suggestion
The two methods `blur()` and `focus()` are inherited from {{domxref("HTMLElement")}}.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few details, and this is good to be merged.

File: files/en-us/web/api/htmlanchorelement/index.md (hunk 4 above)
Comment (estelle) [reply files/en-us/web/api/htmlanchorelement/index.md:93]: It's on the HTMLAnchorElement page, so I left it.

**Comment by estelle** [comment]
Thanks, @teoli2003. There were 3 suggestions that I didn't commit. I realize it's historical data, and generally no need to keep historical information, but did want to point out why I kept it, and leave it up to you if you still think it is not necessary. I am fine either way.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**